### PR TITLE
refactor-introduce

### PR DESCRIPTION
**Pull request by caoergou**
- refactor some introduce page for better typography.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,11 +46,12 @@ If you are accepeted by HM university and pay the deposit to confirm enrolment w
 Each CAS has a unique reference number containing course information and student information.  
 If you want, please email us or open an issue.
 
-## Alumni Mail
+## About  Alumni Mail
```
Comment (KevinZonda) [inline README.md:49]: Single space

File: Characteristic-zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+[English (UK)](Colleges.md) | **简体中文（中国大陆）**
+
+# 家里蹲大学办学特色
+
+## 教学理念
```
Comment (KevinZonda) [inline Characteristic-zh-cn.md:5]: This shouldn't be like this

Following paragraphs are derived from University of Oxford. Maybe get some inspiration from them.

File: README.md [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
-- **About**
-- [What are Home colleges?](Colleges.md)
+## About Home University
 
 Home University, also known as HM University or 家里蹲大学, is a virtual global comprehensive collegiate university.  
-We want to reduce the involution of students and enhance their own abilities.
-We want the definition of success to be less about quantifiable virtual products but more about the students themselves.
+
+
+In order to achieve the following vision, we build the Home University:
```
Comment (KevinZonda) [inline README.md:9]: started not build

Maybe use to in stead of in order to

**Review by KevinZonda — CHANGES_REQUESTED** [review]

File: Characteristic-zh-cn.md (hunk 2 above)
Comment (caoergou) [reply Characteristic-zh-cn.md:5]: would u give me a url about that?

Comment (Cubik65536) [reply Characteristic-zh-cn.md:5]: check ox.ac.uk

**Pull request closed without merging**